Gemfile

```diff
@@ -23,6 +23,9 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 
 gem 'devise'
 gem 'foundation-rails'
+gem 'carrierwave'
+gem 'fog'
+gem 'dotenv-rails'
```

Comment:
Put this in your `group :development, :environment`. Heroku doesn't require `dotenv-rails` to load environment variables.